packages/nextjs/tsconfig.json

```diff
@@ -23,6 +23,6 @@
       }
     ]
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "app/builders/0x28482B1279E442f49eE76351801232D58f341CB9/page.tx"],
```

Comment:
```"app/builders/0x28482B1279E442f49eE76351801232D58f341CB9/page.tx"``` is not needed since the wildcards (**/*.ts and **/*.tsx) will cover all TypeScript files in your project structure. So you might remove it.